lib/win32/certstore/store_base.rb

```diff
@@ -125,11 +111,30 @@ def cert_get_name_args(pcert_context, cert_name)
         [pcert_context, CERT_NAME_FRIENDLY_DISPLAY_TYPE, CERT_NAME_ISSUER_FLAG, nil, cert_name, 1024]
       end
 
+      # Remove extra space and : from thumbprint
+      def update_thumbprint(certificate_thumbprint)
+        certificate_thumbprint.gsub(/[^A-Za-z0-9]/, '')
+      end
+
       # Convert OpenSSL::X509::Certificate object in .der formate
       def der_cert(cert_obj)
         FFI::MemoryPointer.from_string(cert_obj.to_der)
       end
 
+      # Get certificate pem
+      def get_cert_pem(thumbprint)
+        IO::popen(["powershell.exe", cert_ps_cmd(thumbprint)]) {|io| io.readlines}
+      end
+
+      # Format pem
+      def format_pem(cert_pem)
+        cert_pem.join
+      end
+
+      # Build pem to OpenSSL::X509::Certificate object
+      def build_openssl_obj(cert_pem)
+        OpenSSL::X509::Certificate.new cert_pem
```

Comment:
`OpenSSL::X509::Certificate.new(cert_pem)`

https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

Reply:
sorry for that.